src/features/yamlEditor/components/editor.tsx

```diff
@@ -27,7 +27,7 @@ export const CodeEditor = ({ code, onChangeEditor }: ICodeEditor) => {
   const convertToJSON = (e: string) => {
     try {
       setIsError(false);
-      onChangeEditor(parse(e));
+      onChangeEditor(parse(e) || {});
```

Comment:
This is a lazy fix to prevent a runtime error that occurs if you clear the YAML editor. But a more rigorous fix would take a lot more time because the editor right now just doesn't have good validation.